### DISCUSSION TITLE
Library won't validate when trying to use Relink Toolbox.

### DISCUSSION THREAD

**Discussion by tangierc** [latenitefilms/BRAWToolbox#142] [Support]
I am attempting to load my library into the Relink BRAW Clips Toolbox because it's the only way I can hopefully get all of the BRAW clips updated for all of my events and projects without having to repeat the relinking process (particularly for project timelines) multiple times. However when I drag my library into the toolbox I get a validation error shown below.

![Screenshot 2023-06-26 at 5 45 00 PM](https://github.com/latenitefilms/BRAWToolbox/assets/9577806/30b7740c-3dbf-4bb1-be36-0db1e94b3c93)

**Comment by tangierc** [comment]
I tried it again and saw something new I've never seen before:
![Screenshot 2023-06-26 at 5 55 30 PM](https://github.com/latenitefilms/BRAWToolbox/assets/9577806/198ba162-cceb-4e39-9769-a0d4881dd251)

**Reply by latenitefilms** [reply]
Ummm... it's possible it's a Final Cut Pro bug. Can you try export a FCPXML of your library then import it back into a new Final Cut Pro library - does that work?

**Comment by tangierc** [comment]
I’ll give it a try. I’ve had a lot of trouble reimporting the XML that FCP puts out lately. Because of the validation problems, I wasn’t able to use EDL-X or X2Pro.  Da Vinci Resolve was my only way out to prepare media for sound and color teams.

Tangier

**Comment by tangierc** [comment]
Still doesn’t validate. I’ll send feedback to Apple.
